### PR TITLE
Introduce shortnames for CRDs

### LINKED ISSUE #107: Add short-name for steward's PipelineRun resource
Since tekton and stewards share the same resource name `PipelineRun`, it is a bit cumbersome to work with those resources with kubectl and bash completion.

```bash
$ kubectl get pipelineruns.
pipelineruns.steward.sap.com  pipelineruns.tekton.dev
```

Tekton introduces the short-name `pr` for it's PipelineRun resource. It would be a nice enhancement to add a short-name for steward's PipelineRun resource such as `stpr` or similar.

```
$ kubectl api-resources | grep pipelineruns
pipelineruns                                   steward.sap.com                true         PipelineRun
pipelineruns                      pr,prs       tekton.dev                     true         PipelineRun
```

### PR DESCRIPTION
**Pull request by alxsap**
fixes #107

### CODE REVIEW

File: charts/steward/crds/pipelineruns.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ spec:
     kind: PipelineRun
     singular: pipelinerun
     plural: pipelineruns
+    shortNames:
+    - spr
```
Comment (rinckm) [inline charts/steward/crds/pipelineruns.yaml:13]: Why do you use s prefix? I think pr and tn are fine.

File: charts/steward/crds/tenants.yaml [hunk 2]
```diff
@@ -9,6 +9,9 @@ spec:
     kind: Tenant
     singular: tenant
     plural: tenants
+    shortNames:
+    - stn
```
Comment (alxsap) [inline charts/steward/crds/tenants.yaml:13]: `st` seems to be free as well. But chances are higher to conflict with future official resources.

**Comment by alxsap** [comment]
Also add documentation

**Comment by alxsap** [comment]
@anbrsap @holgpar @romanisb Do you agree with the chosen CRD short names?

**Pull request merged**